### DISCUSSION TITLE
SECURITY_SMS_SERVICE

### DISCUSSION THREAD

**Discussion by najibfahs** [pallets-eco/flask-security#459] [Q&A]
hello, I am trying to use the sms authentication method in flask-security-too. I am not clear on the setting of SECURITY_SMS_SERVICE parameter. for instance, should I set it to "at&t" or "sprint" or "verizon"?

**Comment by najibfahs** [comment]
maybe someone can tell me what additional setup I need to make the sms method of 2-fa work. thanks!

**Reply by jwag956** [reply]
You need an SMS provider that can SEND an SMS for you - for example twilio - which is supported out of the box - just install pypi package twilio (and of course you have to sign up with twilio)

**Reply by najibfahs** [reply]
thanks for your response. I would like to suggest additional documentation on this, and maybe a simple working example to help others make it work, on this page: https://flask-security-too.readthedocs.io/en/stable/two_factor_configurations.html. 
At this point, I have been slowly moving forward towards making it work. for instance, based on an error, I realized I needed to install the _phonenumbers_ package...
